taca/nanopore/nanopore.py

```diff
@@ -56,6 +56,39 @@ def transfer_run(self):
             return False
         return True
 
+    def transfer_metadata(self):
+        """Copies run dir (excluding seq data) to metadata dir"""
+
+        try:
+
+            exclude_patterns = [
+                # Main seq dirs
+                "**/bam*/***",
+                "**/fast5*/***",
+                "**/fastq*/***",
+                # Any files found elsewhere
+                "*.bam*",
+                "*.fast5*",
+                "*.fastq*",
+            ]
+
+            exclude_patterns_quoted = [
+                "'" + pattern + "'" for pattern in exclude_patterns
+            ]
+
+            src = self.run_dir
+            dst = os.path.join(self.metadata_dir)
+
+            os.system(
+                f"rsync -rv --exclude={{{','.join(exclude_patterns_quoted)}}} {src} {dst}"
+            )
+
+            return True
+
+        except:
```

Comment:
This would throw away any error messages that would be helpful for troubleshooting. Might be better to try and catch specific errors or at least log the error message.

Reply:
99183fc1769cb2780cab73b90139f9ccd77b6bdb